### PR TITLE
extracted cdc

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -11,9 +11,12 @@ springBootCdcVersion=2.1.1.RELEASE
...
 version=0.15.0-SNAPSHOT
 
-eventuateClientVersion=0.20.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
 
 debeziumVersion=0.3.6
 dropwizardMetricsVersion=3.1.2
 
 eventuateUtilVersion=0.2.0.RELEASE
+eventuateCdcVersion=0.1.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: Is this used? See comments about how it should only be used testCompile and for image versions

File: gradle.properties [hunk 2]
```diff
@@ -11,9 +11,12 @@ springBootCdcVersion=2.1.1.RELEASE
...
+eventuateCommonVersion=0.4.0.RELEASE
+eventuateMessagingKafkaVersion=0.1.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: Please use the .RELEASE version

File: gradle.properties (hunk 1 above)
Comment (dartartem) [reply gradle.properties]: I will fix

File: gradle.properties (hunk 2 above)
Comment (dartartem) [reply gradle.properties]: ok, will do

**Pull request merged**